### PR TITLE
video fixes

### PR DESCRIPTION
**Pull request by loicpoulain**
Fix video buffer alignement
Fix mcux csi buffer dequeueing
Flag polling features

Tested with video/capture and video/tcpserversink samples and mimxrt1064 board

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]
LGTM, not an expert.

File: drivers/video/video_mcux_csi.c [hunk 1]
```diff
@@ -101,10 +103,13 @@ static void __frame_done_cb(CSI_Type *base, csi_handle_t *handle,
 	k_fifo_put(&data->fifo_out, vbuf);
 
 done:
+#ifdef CONFIG_POLL
```
Comment (jfischer-phytec-iot) [inline drivers/video/video_mcux_csi.c]: You can use ` if (IS_ENABLED(CONFIG_POLL))`

Comment (loicpoulain) [reply drivers/video/video_mcux_csi.c]: ack, changed.

Comment (loicpoulain) [reply drivers/video/video_mcux_csi.c]: indeed, thanks, done.

**Review by jfischer-no — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]
thanks @loicpoulain

**Pull request merged**